content/articles/getting-started-with-neural-network-in-pytorch/index.md

```diff
@@ -0,0 +1,287 @@
+Neural networks are an algorithmic set of rules for discovering patterns in data. Using a neural network model will help us understand how our brains process information in the same way. The networks are constructed from discrete pieces that resemble neurons and are often called "neurons" or "units." There are a certain number of weighted inputs for each unit. 
+
+In this tutorial, we'll learn how to use Pytorch to create a neural network from beginning to end.
+
+### Prerequisites
+To get started, you'll need to have:
+- Basic understanding of neural networks.
+- Basic understanding of Pytorch.
+
+### Table of Contents
+- [Introduction to Pytorch](#introduction-to-pytorch)
+- [Creating a neural network](#creating-a-neural-network)
+- [Training the neural network](#training-the-neural-network)
+- [Conclusion](#conclusion)
+
+Nodes in a neural network may be divided into three types, namely:
+- Input nodes - These nodes bring in data from the outside world. The Input nodes do nothing more than transmitting data to the hidden nodes; they do not undertake any computations.
+- Hidden nodes - These nodes are the ones that perform computations. They are the ones that are responsible for learning the weights of the connections between the input nodes and the output nodes. 
+- Output nodes - These nodes are the ones that are responsible for the final output. They are the ones that are responsible for the final decision.
+
+### Introduction to Pytorch
+Pytorch is a [Python](https://pytorch.org/) library that allows us to create neural networks. It is a high-level library that enables us to build neural networks straightforwardly. It is also a prevalent library for deep learning.
+
+#### Install Pytorch
+To install Pytorch, we need to install the [Python](https://www.python.org/) library. We can do this by using the command:
```

Comment:
Here you say we install python but the command below it install, PyTorch, won't this confuse the reader?